backend/commands/import_attendance_list.py

```diff
@@ -0,0 +1,80 @@
+import csv
+import click
+
+from flask.cli import with_appcontext
+from backend.models import Hacknight, Participant
+from backend.extensions import db
+
+
+@click.command()
+@with_appcontext
+def import_attendance_list():
+    """Import attendance list."""
+    rows = []
+    list_of_participants = []
+    users_dict = {}
+
+    # open file from exported csv
+    with open("./temp/list.csv") as csv_file:
+        csv_reader = csv.reader(csv_file, delimiter=",")
+        for row in csv_reader:
+            rows.append(row)
+
+    # get dates of historical hacknights
+    dates = rows[0][4:]
+    click.echo(f"Created list of dates: {dates}")
+
+    # skip header
+    rows = rows[1:]
+    # prepare users' data
+    for item in rows:
+        _id, github, name, phone, email, *values = item
+        first_name = name.split()[0]
+        last_name = name.split()[1]
+
+        users_dict["id"] = _id
+        users_dict["github"] = github
+        users_dict["first_name"] = first_name
+        users_dict["last_name"] = last_name
+        users_dict["email"] = email
+        users_dict["phone"] = phone
+
+        list_of_participants.append(users_dict)
+        users_dict = {}
+
+    # create hacknights
+    for date in dates:
+        splitted_date = date.split(".")
+        formatted_date = f"{splitted_date[2]}-{splitted_date[1]}-{splitted_date[0]}"
+        hacknight = Hacknight(date=formatted_date)
+        db.session.add(hacknight)
```

Comment:
bit more obvious imo, though I'm not sure if it should be `year-month-day` or `day-month-year`).
```suggestion
        year, month, day = date.split(".")
        hacknight = Hacknight(date=f"{year}-{month}-{day}")
        db.session.add(hacknight)
        
```